docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md

```diff
@@ -13,6 +13,34 @@ type: docs
 
 For on-premises deployments of YugabyteDB universes, you need to import nodes that can be managed by YugabyteDB Anywhere.
 
+## Prerequisites
+
+Using YugabyteDB Anywhere (YBA), you can deploy YugabyteDB universes on nodes with the following architectures and operating systems.
+
+### Supported CPU architectures
+
+YBA supports deploying YugabyteDB on both x86 and ARM (aarch64) architecture-based hardware.
+
+Note that support for ARM architectures is unavailable for airgapped setups, because YBA ARM support for [AWS Graviton](https://aws.amazon.com/ec2/graviton/) requires Internet connectivity.
```

Comment:
Where are we getting this from? There is no technical limitation to deploying on ARM in airgap, we just require some different packages. Along with rsync/openssl (always required) they will need
libatomic and glibc-langpack-en (for Fedora based operating systems)
libatomic1 and libncurses6 (for Debian based operating systems)

Reply:
The only ARM limitation is azcopy does not work so you can't backup to Azure Cloud Storage bucket

Reply:
https://github.com/yugabyte/yugabyte-db/pull/16951

Reply:
@chidmuthu should we delete this then

Reply:
Yes, we can remove that part. There is a way for airgapped customers they just will have to manually copy over the DB package.